extension/source/QuillStorageCells.ts

```diff
@@ -19,7 +19,10 @@ import { QuillTransaction } from './types/Rpc';
 // fields that always have concrete values incrementally without breaking
 // existing clients.
 
-function QuillStorageCells(storage: CellCollection) {
+function QuillStorageCells(
+  storage: CellCollection,
```

Comment:
lets make this change to be more explicit about the storage type
```diff
- storage
+ standardStorage
```
A few more places within the file